doc/contributing/reconizing-contributors.md

```diff
@@ -20,6 +20,26 @@ Some of the benefits we hope to achieve through these programs include:
   done by individuals or teams like those mentioned. This should
   encourage other people to get involved and make similar contributions
 
+## Sponsorship link on nodejs/node README.md
+
+Collaborators may add a single link beside their name on the nodejs/node
```

Comment:
```suggestion
Collaborators may add a single link beside their name on the
```

Both this file and the README are in the same repository, so I don't think it needs to be specified explicitly

Reply:
I think it is better to be specific in this case.